imod/protocols/protocol_etomo.py

```diff
@@ -281,50 +276,48 @@ def createOutput(self):
                     self.debug("Slice index is %s" % sliceIndex)
                     newTi.setLocation(sliceIndex, aligFilePath)
                     if tltList is not None:
-                        newTi.setTiltAngle(float(tltList[sliceIndex-1]))
-                    xAli, _, _, _ = ih.getDimensions(newTi.getFileName() + ":mrc")
-                    newTi.setSamplingRate(self.getPixSizeFromDimensions(xAli))
+                        newTi.setTiltAngle(float(tltList[sliceIndex - 1]))
```

Comment:
We do not need to set the samplingRate on the newTi? Which kind of make sense.

Reply:
apparently not. I can see it's set correclty in sqlite for each ti image